packages/angular/src/app/pages/analytics-geography/analytics-geography.component.ts

```diff
@@ -40,6 +40,21 @@ export class AnalyticsGeographyComponent implements OnInit, OnDestroy {
 
   salesByStateMarkers;
 
+  subscription: Subscription = new Subscription();
+
+  constructor(private service: RwaService) {
+  }
+
+  ngOnInit(): void {
+    const dates = analyticsPanelItems[4].value.split('/');
+
+    this.loadData(dates[0], dates[1]);
+  }
+
+  ngOnDestroy(): void {
+    this.subscription.unsubscribe();
```

Comment:
do we really need to unsubscribe from service?

Reply:
Fixed